src/ui/youtube-modal/youtube-modal.ts

```diff
@@ -71,6 +71,11 @@ interface PlayOptions {
    * The time in seconds to offset the video when the video plays.
    */
   startTime?: number;
+
+  /**
+   * YT player orientation. Defaults to "landscape".
+   */
+  orientation?: string;
```

Comment:
Is this just for me to style with CSS when I call it; or does it impact the default player somehow? I traced through `playOptions` usage and didn't see it referenced anywhere.

Reply:
Yeah just for CSS. The YT iframe player will automatically adapt to its orientation.

Reply:
👍 Might be good to clarify in the comment that it just sets the data attribute on the player (rather than impacting anything else). Other than that nit lgtm